modules/thermo-cycler/thermo-cycler-arduino/gcode.cpp

```diff
@@ -0,0 +1,141 @@
+#include "gcode.h"
+
+Gcode::Gcode() {
+}
+
+void Gcode::_strip_serial_buffer() {
+  for(uint8_t i=0;i<SERIAL_BUFFER_STRING.length();i++) {
+    for (uint8_t c=0;c<sizeof(CHARACTERS_TO_STRIP);c++) {
+      if(SERIAL_BUFFER_STRING[i] == CHARACTERS_TO_STRIP[c]) {
+        SERIAL_BUFFER_STRING.remove(i, 1);
+        i--;
+        break;
+      }
+    }
+  }
+}
+
+bool Gcode::pop_command() {
```

Comment:
Do we need this to set a class variable that's polled later? Couldn't we have this return the code?